databases/Assignment2_answers.sql

```diff
@@ -0,0 +1,316 @@
+1. Which laptop models are shown in the Product relation?
+SELECT DISTINCT(Model)
+FROM Product
+WHERE Type = laptop;
+
+2. Find the set of all manufacturers ordered alphabetically.
+SELECT DISTINCT(Manufacturer)
+FROM Product
+ORDER BY DESC; # is there an alpha order?
+# all data types are by default order-able
+
+
+3. Which computer models appear in the database?
+SELECT Model
+FROM Product
+WHERE Type = pc OR Type = laptop;
+
+OR --> (below is a better idea lol)
+
+SELECT DISTINCT(Model)
+FROM Computer;
+
+
+4. Which computer models use 256 MB of RAM?
+SELECT Model
+FROM Computer
+WHERE Computer.RAM = 256;
+
+5. Which computer models cost less than $1100?
+SELECT Model
+FROM Computer
+WHERE Price < 1100;
+
+6. Which laptop models have a HD of at least 50 GB and cost less than $1400?
+SELECT Model
+FROM Computer
+INNER JOIN Product USING Model
+WHERE Product.Type = "laptop" Computer.HD >= 50 AND Price < 1400;
+
+7. Which laser printer models can print color documents?
+SELECT Model
+FROM Printer
+WHERE Printer.Type = laser AND Printer.Color = true;
```

Comment:
clarification: the true here is not the boolean true, the values of the col "Color" are the strings "true" and "false"

does this fix things?
or does your comment still apply?